### PR TITLE
add multi-page tiff support in split file to collection

### PR DESCRIPTION
**Pull request by davelopez**
Added multi-page tiff as a new supported file type in split_file_to_collection tool.
Changed the python required version from 3.5 to 3.8.2 as the ``planemo lint --conda_requirements`` suggested for best practice.

### CODE REVIEW

File: tools/text_processing/split_file_to_collection/split_file_to_collection.xml [hunk 1]
```diff
@@ -396,7 +418,7 @@
                 <element name="test_000001" file="test_1.fastq" ftype="fastq"/>
             </output_collection>
         </test>
-        <!-- splitting of fasta w random assignment and specific filename prefix -->
+        <!-- 12 splitting of fasta w random assignment and specific filename prefix -->
         <test>
```
Comment (bgruening) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.xml:422]: you could enhance all test by using num_expect_outputs https://docs.galaxyproject.org/en/latest/dev/schema.html#id78

Comment (davelopez) [reply tools/text_processing/split_file_to_collection/split_file_to_collection.xml:422]: Cool, thanks! I will update the test to use this!

**Comment by davelopez** [comment]
Well, I can try :)

File: tools/text_processing/split_file_to_collection/split_file_to_collection.xml [hunk 2]
```diff
@@ -565,7 +613,8 @@ The default regular expression uses each value in the column without modifying i
 
 Two modes are available for the tool. For the main mode, the number of output files is selected. In this case, records are shared out between this number of files. Alternatively, 'chunking mode' can be selected, which puts a fixed number of records (the 'chunk size') into each output file.
 
-    ]]></help>
```
Comment (beatrizserrano) [inline tools/text_processing/split_file_to_collection/split_file_to_collection.xml:568]: this needs love

**Comment by beatrizserrano** [comment]
should we close this PR?

**Comment by davelopez** [comment]
Ok for me, I don't think this is relevant any more.

**Pull request closed without merging**